plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -82,6 +99,35 @@ func calculateResourceOccupancy(pod *api.Pod, podRequests *schedulercache.Resour
 	}
 }
 
+// Calculate the resource used on a node.  'node' has information about the resources on the node.
+// 'pods' is a list of pods currently scheduled on the node.
+// TODO: Use Node() from nodeInfo instead of passing it.
+func calculateUsedPriority(pod *api.Pod, podRequests *schedulercache.Resource, node *api.Node, nodeInfo *schedulercache.NodeInfo) schedulerapi.HostPriority {
```

Comment:
Could this function just call calculateUnusedPriority(), and then replace each priority score X with 10-X and re-sort (so they end up in the opposite order as returned from calculateUnusedPriority())? (And then you don't even need calculateUsedScore())